pkg/cvo/status.go

```diff
@@ -211,9 +211,14 @@ func (optr *Operator) syncStatus(original, config *configv1.ClusterVersion, stat
 				LastTransitionTime: now,
 			})
 		} else {
-			message := fmt.Sprintf("Working towards %s", version)
-			if len(validationErrs) > 0 {
+			var message string
+			switch {
+			case len(validationErrs) > 0:
 				message = fmt.Sprintf("Reconciling %s: the cluster version is invalid", version)
+			case status.Fraction > 0:
+				message = fmt.Sprintf("Working towards %s: %.0f%%", version, status.Fraction*100)
```

Comment:
just a nit: 
"Working towards %s: %.0f%% complete" or something like that ?
because `Working towards 4.0.1: 45%` seems incomplete.

Reply:
Agree, needs something.  Will play with complete and some other variants

Reply:
Complete seemed to work fine, we can tweak later if we come up with something better.  Updated.